assembly/src/main/descriptors/kapua-broker.xml

```diff
@@ -167,7 +167,11 @@
                 <include>org.apache.shiro:shiro-core</include>
 
                 <include>org.elasticsearch:elasticsearch</include>
+                <include>org.elasticsearch.client:transport</include>
                 <include>org.elasticsearch:securesm</include>
+                <include>org.slf4j:slf4j-api</include>
+                <include>org.slf4j:log4j-over-slf4j</include>
+                <include>org.apache.logging.log4j:log4j-to-slf4j</include>
```

Comment:
I think that with 5.3 and on it is kind of requirement log4j or bridge from it.
https://www.elastic.co/guide/en/elasticsearch/client/java-api/5.3/_using_another_logger.html